### DISCUSSION TITLE
Potential bug in OpenRoad using mulithread

### DISCUSSION THREAD

**Discussion by jsaenznoval** [The-OpenROAD-Project/OpenROAD#6354] [Q&A]
Hi all
When using the `-threads` option with thopenroad to specify a high number of threads (e.g., 12), the OpenROAD GUI hangs when attempting to generate timing reports, either by clicking the timing report button or using the `report_checks` command.
`openroad -threads 12 <tcl_script>.tcl`

Version: v2.0-11183-g2f133cf95

There was a bug related and already reported?
Is this happening for other users?
Thanks

**Comment by maliberty** [comment]
Please open an issue with a test case.  Does it only happen with 12 threads?
